src/helpers/urlUtils.ts

```diff
@@ -93,29 +98,33 @@ export const getLinkFromText = (text: string): string | undefined => {
 };
 
 export const getHttpLinkFromText = (text: string): string | undefined => {
-    const httpLink = text.match(/(http.?:\/\/.*?)( |$)/);
+    const httpLink = text.match(HTTP_URL_MATCHER);
     if (httpLink != null) {
         return httpLink[1];
     }
     return undefined;
 };
 
 export const getBzzFeedLinkFromText = (text: string): string | undefined => {
-    const bzzFeedLink = text.match(/(bzz-feed:\/\?user=0x[a-f0-9]{40})( |$)/);
+    const bzzFeedLink = text.match(BZZ_FEED_MATCHER);
     if (bzzFeedLink != null) {
         return bzzFeedLink[1];
     }
     return undefined;
 };
 
 export const getBzzLinkFromText = (text: string): string | undefined => {
-    const bzzLink = text.match(/(bzz:\/\/[a-f0-9]{64})( |$)/);
+    const bzzLink = text.match(BZZ_RESOURCE_MATCHER);
     if (bzzLink != null) {
         return bzzLink[1];
     }
     return undefined;
 };
 
+export const isFelfeleResource = (text: string): boolean => {
```

Comment:
We don't use `BZZ` resources as links, only `BZZ_FEED` resources.

Reply:
fixed in 15b9c82